### PR TITLE
Update the content of the `Using Object.keys() on primitives` section

### PR DESCRIPTION
**Pull request by codersjj**
### Description

Update the content of the `Using Object.keys() on primitives` section.

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/object/keys/index.md [hunk 1]
```diff
@@ -68,16 +68,21 @@ If you want _all_ string-keyed own properties, including non-enumerable ones, se
 
 ### Using Object.keys() on primitives
 
-Non-object arguments are [coerced to objects](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#object_coercion). Only strings may have own enumerable properties, while all other primitives return an empty array.
+Non-object arguments are [coerced to objects](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#object_coercion). Only strings may have own enumerable properties, while all other primitives(except [`undefined`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined) and [`null`](/en-US/docs/Web/JavaScript/Reference/Operators/null), which will throw a {{jsxref("TypeError")}}) return an empty array.
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/object/keys/index.md:71]:
```suggestion
Non-object arguments are [coerced to objects](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object#object_coercion). [`undefined`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined) and [`null`](/en-US/docs/Web/JavaScript/Reference/Operators/null) cannot be coerced to objects and throw a {{jsxref("TypeError")}} upfront. Only strings may have own enumerable properties, while all other primitives return an empty array.
```

File: files/en-us/web/javascript/reference/global_objects/object/keys/index.md [hunk 2]
```diff
@@ -68,16 +68,21 @@ If you want _all_ string-keyed own properties, including non-enumerable ones, se
...
-// Other primitives have no own properties
+// Other primitives except undefined and null have no own properties
 console.log(Object.keys(100)); // []
 ```
 
+```js example-bad
+console.log(Object.keys(undefined)); // TypeError: Cannot convert undefined or null to object
+console.log(Object.keys(null)); // TypeError: Cannot convert undefined or null to object
+```
+
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/object/keys/index.md:85]:
```suggestion
```

**Review by Josh-Cena — APPROVED** [review]
Sure, thanks. I don't think it's worth splitting hairs about it too much (because the error is thrown during object coercion, not property enumeration), but it's better to be precise.

**Comment by codersjj** [comment]
> Sure, thanks. I don't think it's worth splitting hairs about it too much (because the error is thrown during object coercion, not property enumeration), but it's better to be precise.

You're right. Thank you for the feedback.

**Pull request merged**